runtime/src/chpl-file-utils.c

```diff
@@ -221,7 +221,7 @@ qioerr chpl_fs_is_mount(int* ret, const char* name) {
         // While we haven't found the end of the path (in nextTok)
         strncat(parent, "/", 1);
         // Restore the lost path separator.
-        strncat(parent, curTok, strlen(curTok));
+        strncat(parent, curTok, strlen(curTok) + 1);
```

Comment:
Since the size limit is given as the length of the src, not how much space is left in the dst, isn't this just a roundabout way of writing `strcat(parent, curTok)`?